qemu/tests/cfg/fio_perf.cfg

```diff
@@ -16,29 +16,24 @@
         order_list = "Block_size Iodepth Threads BW(MB/S) IOPS Latency(ms) Host_CPU BW/CPU KVM_Exits Util%"
         guest_ver_cmd = "uname -r"
         check_install_fio = "which fio"
-        tarball = "performance/fio-3.27.tar.gz"
-        fio_path = "/tmp/fio-3.27"
-        compile_cmd = "make && make install"
         pre_cmd = "i=`/bin/ls /dev/[vs]db` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt"
         ppc64, ppc64le:
             virtio_blk:
                 pre_cmd = "i=`/bin/ls /dev/vda` && mkfs.xfs $i > /dev/null; partprobe; umount /mnt; mount $i /mnt"
         drop_cache = "sync && echo 3 > /proc/sys/vm/drop_caches"
         guest_result_file = /tmp/fio_result
-        fio_cmd = "fio --rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/mnt/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=/tmp/fio_result &> /dev/null"
+        fio_options = "--rw=%s --bs=%s --iodepth=%s --runtime=1m --direct=1 --filename=/mnt/%s --name=job1 --ioengine=libaio --thread --group_reporting --numjobs=%s --size=512MB --time_based --output=/tmp/fio_result &> /dev/null"
```

Comment:
Suggest replace the hardcode `--output=/tmp/fio_result` with `${guest_result_file}`, and same for windows guest.
Then it is more clear in the script for the clean step.